medcat/config_meta_cat.py

```diff
@@ -5,6 +5,7 @@
 class General(MixingConfig, BaseModel):
     """The General part of the MetaCAT config"""
     device: str = 'cpu'
+    """Device to used by the module"""
```

Comment:
add in this link this: https://pytorch.org/docs/stable/tensor_attributes.html#torch.device

Reply:
done!